app/models/organization.rb

```diff
@@ -10,26 +11,22 @@ def with_transfer
     end
   end
   has_many :all_transfers, class_name: "Transfer", through: :all_movements, source: :transfer
-
   has_one :account, as: :accountable
-  after_create :create_account
-
   has_many :member_accounts, through: :members, source: :account
-
   has_many :posts
   has_many :offers
   has_many :inquiries
-
   has_many :documents, as: :documentable
 
-  BOOTSWATCH_THEMES = %w[amelia cerulean cosmo cyborg flatly journal readable
-                         simplex slate spacelab united]
-  # validates :theme, allow_nil: true, inclusion: {in: BOOTSWATCH_THEMES}
-
   scope :matching, ->(str) {
     where(Organization.arel_table[:name].matches("%#{str}%"))
   }
 
+  validates_uniqueness_of :name
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.